website/docs/index.html.markdown

```diff
@@ -208,6 +208,8 @@ Nested `endpoints` block supports the following:
 
 * `location` - (Optional) Use this to override the default endpoint URL constructed from the `region`. It's typically used to connect to custom Location Service endpoints.",
 
+* `cas` - (Optional) Use this to override the default endpoint URL constructed from the `region`. It's typically used to connect to custom CAS endpoints.
```

Comment:
Please add `cas_endpoint` in the provider.go.

Reply:
done